modules/net-lb-app-ext-regional/variables.tf

```diff
@@ -142,6 +142,13 @@ variable "neg_configs" {
   }
 }
 
+variable "network_tier_standard" {
+  description = "Use standard network tier."
+  type        = bool
+  nullable    = false
+  default     = true
```

Comment:
Should this default to false? Provider assumes [PREMIUM](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_forwarding_rule#network_tier-1)

And there is also [default network tier](https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/compute_project_default_network_tier) which will be incompatible with this, but explicit is better than implicit :-)

Reply:
THis is the default we had hardcoded, so the we are keeping compatibility. It also makes sense to use the lower tier for regional.

![image](https://github.com/user-attachments/assets/fcd1ea38-2bd4-4153-b194-c00cdbf5106c)

Reply:
And there's only two supported tiers, so this needs to be a `bool`. :)